Source/Plugin.BLE/Windows/Adapter.cs

```diff
@@ -144,11 +144,8 @@ private void Device_ConnectionStatusChanged(BluetoothLEDevice nativeDevice, obje
                 }
 
                 // fire the correct event (DeviceDisconnected or DeviceConnectionLost)
-                HandleDisconnectedDevice(isNormalDisconnect, disconnectedDevice);
-                if (isNormalDisconnect)
-                {
-                    nativeDevice.ConnectionStatusChanged -= Device_ConnectionStatusChanged;
-                }
+                HandleDisconnectedDevice(isNormalDisconnect, disconnectedDevice);                
```

Comment:
There is no real change in line 147. Apparently you're adding unnecessary whitespace at the end.